protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -598,6 +603,77 @@ message PositionUpdates{
     repeated vega.Position positions = 1;
 }
 
+message LedgerEntryFilter {
+    bool close_on_account_ids = 1;
+    repeated string account_from_ids = 2;
+    repeated string account_to_ids = 3;
+    bool close_on_account_filters = 4;
+    repeated AccountFilter account_from_filters = 5;
+    repeated AccountFilter account_to_filters = 6;
+    bool close_on_account_ids_filters = 7;
+    bool close_on_accounts_transfer_type = 8;
```

Comment:
The difference is that `close_on_transfer_types` closed under the set type of transfer types for from/to accounts and `close_on_accounts_transfer_type` closes that set under the set returned from `accounts_from_filters+accounts_to_filters` + `accountFromID`+`accountToId`.
But since the implementation will be much simplified now, we are going to have:
AccountFrom+AccountTo - as account filters - closed or not
TranferType - a single settings for transfer type and it will be closed or not under the accounts filtering, and that would be it - 3 fileds, 2 booleans